src/main/java/de/uos/informatik/ko/rcp/geneticAlgo/GeneratePop.java

```diff
@@ -0,0 +1,101 @@
+package de.uos.informatik.ko.rcp.geneticAlgo;
+
+import java.lang.reflect.Array;
+import java.lang.Integer;
+import java.util.*;
+import de.uos.informatik.ko.rcp.Utils;
+import de.uos.informatik.ko.rcp.Instance;
+
+
+public class GeneratePop {
+
+    private GeneratePop() {}
+    //change to Hashset of ArrayLists
+    public static HashSet<ArrayList<Integer>> generatePop(Instance instance, Integer pop_size,Random myGenerator){
+        // all activities without predecessor are possible candidates
+        HashSet<ArrayList<Integer>> results = new HashSet<ArrayList<Integer>>();
+        HashMap<Integer,HashSet<Integer>> mypredecessors;
+        HashSet<Integer> candidates = new HashSet<Integer>();
+        //scheduled activities
+        ArrayList<Integer> scheduled = new ArrayList<Integer>();
+        for (int i=0; i< pop_size*5; i++) {
```

Comment:
Where does the `5` come from?

Reply:
Sorry, magic number.
We should use of course use a variable
The loop defines the number of the attempts to create a new population member.
Since the function could generate the same candidate, we have to investigate a appropriate amount of iterations.

Reply:
Right, I didn't consider the possibility of creating the same population member twice. I also think that finding a good number of attempts might require some experimentation